File: Python/Assignments/Module-4/Q5.py
f=open(r'C:\Users\jaish\OneDrive\Desktop\TOPS FILES\Jaishv_SE\Python\Assignments\Module-3\Timepass.txt',"r")
l=f.readlines()
n=int(input("Enter the number of lines you want to read from end : "))
d=len(l)
if n<=d:
    for i in range (d-1,d-n-1,-1):
        print(l[i]) 
else:
    print("The given number exceeds the number of lines in the file.")
    f.close()

File: Assignments/Module 3/Topic 3/Q5.c
#include<stdio.h>

main(){
	int i,j=1,n;
	printf("Enter the number whose factorial you want to print : ");
	scanf("%d",&n);
	for(i=1;i<=n;i++){
		j*=i;
	}
	printf("The factorial of %d is : %d",n,j);
}
